tailwind.config.js

```diff
@@ -72,24 +64,23 @@ module.exports = {
         strict: true,
       }
     ),
+    createThemeFonts({
+      forest: {
+        body: '"Tenor Sans", sans-serif',
+        display: '"Merienda", cursive',
+      },
+      racer: {
+        body: "Montserrat",
+        display: "Anton",
+      },
+      "retro-diner": {
```

Comment:
@eprisr I love seeing this come to action.
In your next round, can you add PatchWrk, NYIAS, DragonCon, Grey, and set grey as default, 
keeping Forest, Race, RetroDiner in there.